File: hotel_api/hotel_data/dto/PersonDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace hotel_data.dto
{
    public class PersonDto
    {

        public long? personID { get; set; }
        public string name { get; set; }
        public string phone { get; set; }
        public string address { get; set; }

        public PersonDto(long? personID, string name, string phone, string address)
        {
            this.personID = personID;
            this.name = name;
            this.phone = phone;
            this.address = address;
        }

    }
}

File: hotel_mobile/app/src/main/java/com/example/hotel_mobile/Dto/PersonDto.kt
package com.example.hotel_mobile.Dto

import com.example.hotel_mobile.services.kSerializeChanger.LocalDateTimeKserialize
import com.example.hotel_mobile.services.kSerializeChanger.UUIDKserialize
import kotlinx.datetime.LocalDateTime
import kotlinx.serialization.Serializable
import java.util.UUID

@Serializable
data class PersonDto(
    val personID: String? = null,
    val name: String,
    val email: String,
    val phone: String,
    val address: String,
    @Serializable(with = LocalDateTimeKserialize::class)
    val createdAt: LocalDateTime? = null
)
